alacritty/build.rs

```diff
@@ -13,6 +13,10 @@
 // limitations under the License.
 
 fn main() {
-    let hash = rustc_tools_util::get_commit_hash().expect("couldn't get commit hash");
+    let mut hash = rustc_tools_util::get_commit_hash().unwrap_or("".to_owned());
+    if hash.is_empty() {
+        hash = std::env::var("GIT_HASH").expect("couldn't get commit hash");
+    }
```

Comment:
If `GIT_HASH` is set at build time, wouldn't it also be set at compile time? So you might be able to just check if the variable is set at all here.